setup.py

```diff
@@ -0,0 +1,25 @@
+import setuptools
+
+with open("README.md", "r", encoding="utf-8") as f:
+    long_description = f.read()
+
+__version__ = "1.0.0"
```

Comment:
why is version defined in __init__.py and in setup.py?

Reply:
This is a bit of an unfortunate hack to work around the following scenario:

* someone installs planetterp and does not have the requests module
* pip goes to install planetterp, which involves importing setup.py 
* `import setup` executes the import statements at the top of setup.py, which in turn execute our `import requests` line in `PlanetTerp.py`. This errors because the user doesn't have `requests` and the install fails
* the solution to this is either to have a `version.py` file and parse that manually to get the version (this is what I do in other projects) or define it in multiple places like we have here.

Further reading: https://stackoverflow.com/a/7071358

Reply:
To clarify, if we wanted it in one place we would put it in `__init__.py`, which means setup.py has to import it from init. This sets off the chain reaction of imports that ends in attempting, and failing, to import requests.

Reply:
I don't fully understand the issue, but I don't like that we need to update the version in two places. I'm ok with a version.py file, at least for now.

Reply:
that's also my preferred resolution, will go ahead with that